src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-credentialsId.html

```diff
@@ -1,5 +1,7 @@
 <div>
-    The list of credentials used to authenticate with Bitbucket Server.<br>
-    Bitbucket Server supports username and password credentials, as well as secret word credentials. When adding new
-    credentials, only these types should be used (other credential types will not appear in the list.)
+    Credentials are required for Jenkins to authenticate with Bitbucket. Depending on the credentials used, they give Jenkins
+    access to some or all of the repositories on your Bitbucket instance, and allow it to create builds. To provide them:<br>
+    1. Select credentials for your instance from the dropdown list (if they exist)<br>
+    2. Add your own credentials for your instance<br>
+    The credentials you provide will determine which Bitbucket instances appear below.
```

Comment:
That is not entirely correct. The dropdown is always populated with the entire list of servers, the credentials _may_ or _may not_ work for a particular instance. 

I think this whole text needs a slight polish as it is not entirely clear how things hang together. How about
```html
Credentials are required for Jenkins to authenticate with Bitbucket. Depending on the credentials used, they give Jenkins access to some or all of the repositories on your Bitbucket instance, and allow it to check out source code for building. To provide them:<br>
<ul>
    <li>
        Select credentials for your instance from the dropdown list (if they exist).
        <br>
        or
    </li>
    <li>Use the credentials provided for the instance (select no credentials to fall back to the instance defined credentials should they exist).</li>
   
</ul>
The credentials you provide will determine which projects and repositories can be cloned.
```

Reply:
Thanks @dkjellin . I've made some updates for the next round of UI changes.